modules/server/src/main/ts/bedrock/server/Apis.ts

```diff
@@ -119,7 +122,26 @@ export const create = (master: DriverMaster | null, maybeDriver: Attempt<any, Br
     pageHasLoaded = true;
   };
 
-  const c = Controller.create(stickyFirstSession, singleTimeout, overallTimeout, testfiles, loglevel);
+  const c = Controller.create(stickyFirstSession, overallTimeout, testfiles, loglevel);
+
+  const maybeSendKeepAlive: () => Promise<void> = (() => {
+    let lastKeepAlive = Date.now();
+    /*
+     * If we're within a minute of the remote driver timeout, send a keep-alive.
+     *
+     * In theory this is only required if no other action has been taken in the last x minutes,
+     * but this happens so rarely it should be fine.
+     */
+    const keepAliveTimer = (Math.max(120, REMOTE_IDLE_TIMEOUT_SECONDS - 60)) * 1000;
```

Comment:
```suggestion
    const keepAliveTimer = Math.max(120, REMOTE_IDLE_TIMEOUT_SECONDS - 60) * 1000;
```

Reply:
WebStorm used to have a setting to identify optimisations like this. I wonder why it's not on by default.